ci-operator/step-registry/cucushift/hypershift-extended/enable-dns/custom-kubeconfig/cucushift-hypershift-extended-enable-dns-custom-kubeconfig-commands.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if [ ! -f "${SHARED_DIR}/kubeconfig" ]; then
+    exit 1
+fi
+export KUBECONFIG="${SHARED_DIR}/kubeconfig"
+if [[ -f "${SHARED_DIR}/proxy-conf.sh" ]]; then
+    source "${SHARED_DIR}/proxy-conf.sh"
+fi
+
+CLUSTER_NAME=$(oc get hostedclusters -n "$HYPERSHIFT_NAMESPACE" -o jsonpath='{.items[0].metadata.name}')
```

Comment:
for agent platform, this hcp is not in the ns "clusters", it's better to check how they set the hcp ns there